src/main/java/be/seeseemelk/mockbukkit/ServerSettingsMock.java

```diff
@@ -0,0 +1,313 @@
+			ServerSettingsMock.this.setAnimalSpawnLimit(getInt("animalSpawnLimit", animalSpawnLimit));
+			ServerSettingsMock.this.setWaterAnimalSpawnLimit(getInt("waterAnimalSpawnLimit", waterAnimalSpawnLimit));
+			ServerSettingsMock.this.setWaterAmbientSpawnLimit(getInt("waterAmbientSpawnLimit", waterAmbientSpawnLimit));
+			ServerSettingsMock.this.setAmbientSpawnLimit(getInt("ambientSpawnLimit", ambientSpawnLimit));
+			ServerSettingsMock.this.setTicksPerAnimalSpawns(getInt("ticksPerAnimalSpawns", ticksPerAnimalSpawns));
+			ServerSettingsMock.this.setTicksPerMonsterSpawns(getInt("ticksPerMonsterSpawns", ticksPerMonsterSpawns));
+			ServerSettingsMock.this.setTicksPerWaterSpawns(getInt("ticksPerWaterSpawns", ticksPerWaterSpawns));
+			ServerSettingsMock.this.setTicksPerWaterAmbientSpawns(getInt("ticksPerWaterAmbientSpawns", ticksPerWaterAmbientSpawns));
+			ServerSettingsMock.this.setTicksPerAmbientSpawns(getInt("ticksPerAmbientSpawns", ticksPerAmbientSpawns));
+		}
+	};
+
+	public void load(File file) throws IOException, InvalidConfigurationException
+	{
+		yamlConfiguration.load(file);
+	}
+
+	public void save(File file) throws IOException
+	{
+		yamlConfiguration.save(file);
+	}
+
+	@Override
+	public boolean equals(Object o)
+	{
+		if (this == o)
+		{
+			return true;
+		}
+
+
+		if (o == null || getClass() != o.getClass())
+		{
+			return false;
+		}
+
+		ServerSettingsMock that = (ServerSettingsMock) o;
+
+		return new EqualsBuilder()
+				.append(allowEnd, that.allowEnd)
+				.append(connectionThrottle, that.connectionThrottle)
+				.append(monsterSpawnLimit, that.monsterSpawnLimit)
+				.append(animalSpawnLimit, that.animalSpawnLimit)
+				.append(waterAnimalSpawnLimit, that.waterAnimalSpawnLimit)
+				.append(waterAmbientSpawnLimit, that.waterAmbientSpawnLimit)
+				.append(ambientSpawnLimit, that.ambientSpawnLimit)
+				.append(ticksPerAnimalSpawns, that.ticksPerAnimalSpawns)
+				.append(ticksPerMonsterSpawns, that.ticksPerMonsterSpawns)
+				.append(ticksPerWaterSpawns, that.ticksPerWaterSpawns)
+				.append(ticksPerWaterAmbientSpawns, that.ticksPerWaterAmbientSpawns)
+				.append(ticksPerAmbientSpawns, that.ticksPerAmbientSpawns)
+				.append(updateFolder, that.updateFolder)
+				.append(worldContainer, that.worldContainer)
+				.append(shutdownMessage, that.shutdownMessage).isEquals();
+	}
+
+	@Override
+	public int hashCode()
+	{
+		return new HashCodeBuilder(17, 37)
+				.append(allowEnd)
+				.append(updateFolder)
+				.append(worldContainer)
+				.append(connectionThrottle)
+				.append(shutdownMessage)
+				.append(monsterSpawnLimit)
+				.append(animalSpawnLimit)
+				.append(waterAnimalSpawnLimit)
+				.append(waterAmbientSpawnLimit)
+				.append(ambientSpawnLimit)
+				.append(ticksPerAnimalSpawns)
+				.append(ticksPerMonsterSpawns)
+				.append(ticksPerWaterSpawns)
+				.append(ticksPerWaterAmbientSpawns)
+				.append(ticksPerAmbientSpawns).toHashCode();
+	}
+
+	public boolean isAllowEnd()
+    {
+		return allowEnd;
+	}
```

Comment:
The indentation here and in the rest of the file is a bit all over the place. Mixing tabs and spaces perhaps?